test/integration/JestTestRunner.it.ts

```diff
@@ -7,165 +7,188 @@ chai.use(chaiAsPromised);
 let expect = chai.expect;
 
 describe('JestTestRunner', function () {
-    let sut: JestTestRunner;
-    this.timeout(10000);
-
-    const expectToHaveSuccessfulTests = (result: RunResult, n: number) => {
-        expect(result.tests.filter(t => t.status === TestStatus.Success)).to.have.length(n);
-    };
-
-    const expectToHaveFailedTests = (result: RunResult, expectedFailureMessages: string[]) => {
-        const actualFailedTests = result.tests.filter(t => t.status === TestStatus.Failed);
-        expect(actualFailedTests).to.have.length(expectedFailureMessages.length);
-        actualFailedTests.forEach(failedTest => expect(failedTest.failureMessages[0]).to.contain(expectedFailureMessages.shift()));
-    };
-
-    function file(overrides?: Partial<FileDescriptor>): FileDescriptor {
-      return Object.assign({}, {
-        name: 'file.js',
-        transpiled: true,
-        included: true,
-        mutated: true,
-        kind: FileKind.Text
-      }, overrides);
-    }
-
-    describe('when all tests succeed', () => {
-        let testRunnerOptions: RunnerOptions;
-
-        before(() => {
-            testRunnerOptions = {
-                files: [
-                    file({ name: 'testResources/sampleProject/src/Add.js', mutated: true }),
-                    file({ name: 'testResources/sampleProject/src/__tests__/AddSpec.js', mutated: false })],
-                port: 9877,
-                strykerOptions: { logLevel: 'trace' }
-            };
-        });
+  let sut: JestTestRunner;
```

Comment:
Do we really need to re-format the whole file because of an `r`? If it must be re-formatted, could you _please_ do that in a separate commit?

Reply:
The `r` change was a different file. I'm adding an integration test here. Also: an indent 2 is in `.editorconfig`, so i think thats why my editor forced this reformatting. Shall i undo the formatting for this file?